google/ggrc-core
# GGRC-2753 New assessments created do not contain date and time for assessments in Notifications

# Issue description

Notification "New assessments were created" doesn't contain date and time for assessments in /_notifications/show_daily_digest

# Steps to test the changes

1. Create new Assessment on the audit page
2. Go to https://ggrc-qa.appspot.com/_notifications/show_daily_digest or https://ggrc-qa.appspot.com/_notifications/show_pending
3. Look at the "New assessments were created" notification
Actual Result: Notification "New assessments were created" doesn't contain date and time for assessments in /_notifications/show_daily_digest
Expected Result: Notification "New assessments were created" should contain date and time for assessments in /_notifications/show_daily_digest

# Solution description

Added the code changes to display the date and time when the new Assessment has been created.
Please find the screenshot for the new changes made:
![GGRC-2753](https://user-images.githubusercontent.com/48320915/59426637-4e801980-8df6-11e9-9f5f-6cb3d0134b6f.png)

# Sanity checklist




# PR Review checklist
